doc/api/cli.md

```diff
@@ -2647,7 +2683,9 @@ done
 [debugger]: debugger.md
 [debugging security implications]: https://nodejs.org/en/docs/guides/debugging-getting-started/#security-implications
 [emit_warning]: process.md#processemitwarningwarning-options
+[environment_variables]: #environment-variables
 [filtering tests by name]: test.md#filtering-tests-by-name
+[ini file format]: https://en.wikipedia.org/wiki/INI_file#Format
```

Comment:
I find this reference confusing:
- We don't support `[section]`
- We don't support `; comment`

If this is just here to justify `key=value`, I think we shouldn't have it.